components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -234,6 +234,14 @@ class BrowserToolbar @JvmOverloads constructor(
         displayToolbar.addNavigationAction(action)
     }
 
+    /**
+     * See [Toolbar.setBackgroundColor]
+     */
+    override fun setBackgroundColor(color: Int) {
+        displayToolbar.setBackgroundColor(color)
+        editToolbar.setBackgroundColor(color)
+    }
+
```

Comment:
This shouldn't be needed as BrowserToolbar is already a View. I had the same problem with EngineView which is just an interface but sometimes I want to use it as a View. I added this helper to it:

```Kotlin
fun asView(): View = this as View
```

How about we add something like that to Toolbar? With that we do not need to mirror all View methods that we may want to access.

Reply:
OK, looks good to me.